google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -272,23 +244,55 @@ Formatter getFormatterProperty(String name, Formatter defaultValue) {
    */
   Logging getLogging() {
     if (logging == null) {
-      logging = options.getService();
+      synchronized (this) {
```

Comment:
Is this change made so that `getLogging` can be called from multiple threads? If so, there's a subtle bug here. A way to fix it is to make `logging` `volatile`. See http://www.cs.umd.edu/~pugh/java/memoryModel/DoubleCheckedLocking.html